base/test/BUILD.gn

```diff
@@ -300,6 +300,12 @@ static_library("test_support") {
     ]
   }
 
+  if (!use_blink && is_starboard) {
+    sources += [
+      "multiprocess_test.cc",
```

Comment:
Starboard is always single-process, so I don't think we should be running this test